components/lightbox/template-preview.css

```diff
@@ -0,0 +1,51 @@
+/**
+ * Copyright 2016 The AMPStart Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+@import '../../css/core-vars';
+
+.ampstart-iphone-mask {
+  background: var(--color-black);
+}
+
+.ampstart-iphone-frame {
+  border: solid 18px #e5ebee;
+  box-shadow: 0px 0px 0px 7px #bfc8ca;
+  border-radius: 15px;
+  width: 360px;
+  height: 630px;
+  padding: 50px 0;
+  background: #fff;
+  top: 50%;
+  left: 50%;
+  margin: -315px 0 0 -180px;
+}
+
+.ampstart-iphone-frame::before,
+.ampstart-iphone-frame::after {
+  content: '';
+  display: block;
+  height: 50px;
+  width: 100%;
+  background: #e5ebee;
```

Comment:
This would not be a published component , it is used for creating a phone like enclosure around our templates being showcased, this does not change with a theme.